lib/rust/src/api.rs

```diff
@@ -9,6 +9,46 @@ pub fn handler<T: 'static>(webview: &mut WebView<'_, T>, arg: &str) -> bool {
     Err(_) => false,
     Ok(command) => {
       match command {
+        Init {} => {
+          webview
+            .handle()
+            .dispatch(move |_webview| {
+              _webview
+                .eval(&format!(
+                  "window['{queue}'] = [];
+                  window['{fn}'] = function (payload, salt, ignoreQueue) {{
+                    window.tauri.promisified({{
+                      cmd: 'validateSalt',
+                      salt: salt
+                    }}).then(function () {{
```

Comment:
then is es6, no?
this means we should probably rewrite this whole thing with es5 style callbacks, no?